File: 프로그래머스/unrated/181939. 더 크게 합치기/더 크게 합치기.py
def solution(a, b):
    # f-string으로 ab, ba 조합을 만든 후 int로 숫자로 바꾼다. max 함수로 크기를 비교한다.
    return max(int(f'{a}{b}'), int(f'{b}{a}'))

File: 프로그래머스/unrated/181939. 더 크게 합치기/더 크게 합치기.js
function solution(a, b) {
    // a, b를 각각 문자열로 바꿔 +로 합친 뒤 다시 숫자로 바꿔 크기를 비교한다.
    const strA = String(a)
    const strB = String(b)
    const ab = Number(strA + strB)
    const ba = Number(strB + strA)
    
    return ab > ba ? ab : ba
}